id:270
user_name:172.26.40.155
user_id:172.26.40.1-fd619819a3174031
script:1 - Pump Control
start_time:1552835730022
command:/eve/scripts/webui-sc/mpumptimed.py all Drug,Media,Waste 1
exit_code:1
>>>>>  OUTPUT STARTED <<<<<
Traceback (most recent call last):
  File "/eve/scripts/webui-sc/mpumptimed.py", line 43, in <module>
    gpioe.append(adafruit_mcp230xx.MCP23017(i2c, address=add))
  File "/usr/local/lib/python3.5/dist-packages/adafruit_mcp230xx.py", line 268, in __init__
    self.iodir = 0xFFFF
  File "/usr/local/lib/python3.5/dist-packages/adafruit_mcp230xx.py", line 350, in iodir
    self._write_u16le(_MCP23017_IODIRA, val)
  File "/usr/local/lib/python3.5/dist-packages/adafruit_mcp230xx.py", line 288, in _write_u16le
    i2c.write(_BUFFER, end=3)
  File "/usr/local/lib/python3.5/dist-packages/adafruit_bus_device/i2c_device.py", line 112, in write
    self.i2c.writeto(self.device_address, buf, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/busio.py", line 64, in writeto
    return self._i2c.writeto(address, memoryview(buffer)[start:end], stop=stop)
  File "/usr/local/lib/python3.5/dist-packages/adafruit_blinka/microcontroller/generic_linux/i2c.py", line 38, in writeto
    self._i2c_bus.write_bytes(address, buffer[start:end])
  File "/usr/local/lib/python3.5/dist-packages/Adafruit_PureIO/smbus.py", line 244, in write_bytes
    self._device.write(buf)
OSError: [Errno 121] Remote I/O error
